brainsets/cli.py

```diff
@@ -31,7 +31,8 @@ def cli():
 @cli.command()
 @click.argument("dataset", type=click.Choice(DATASETS, case_sensitive=False))
 @click.option("-c", "--cores", default=4, help="Number of cores to use")
-def prepare(dataset, cores):
+@click.option("-v", "--verbose", is_flag=True, default=False)
+def prepare(dataset, cores, verbose):
```

Comment:
I used for debugging locally, but seems like this flag is good to keep for future debugs.